delete_dashboards_by_text_in_title/delete_dashboards.py

```diff
@@ -0,0 +1,45 @@
+from datadog import initialize, api
+
+
+def delete_dashboards_by_text_in_title():
+
+    api_key = input("Please enter your API key: ")
+    app_key = input("Please enter your APPLICATION key: ")
+
+    options = {"api_key": api_key, "app_key": app_key}
+
+    initialize(**options)
+
+    text_to_search_in_dashboard = input(
+        "Please enter the text to search in your dashboard titles: "
+    )
+
+    dashboards = api.Dashboard.get_all()["dashboards"]
+
+    dashboards_to_delete = {}
+
+    for dashboard in dashboards:
+        if text_to_search_in_dashboard in dashboard["title"]:
+            dashboards_to_delete[dashboard["id"]] = dashboard["title"]
+
+    dashboard_list = dashboards_to_delete.values()
+
+    if len(dashboard_list) == 0:
+        print("There were no dashboards found for this query.")
+    else:
+        print("Dashboards to delete:\n" + "\n".join(dashboard_list))
+
+        delete_or_not = input(
```

Comment:
Default to no instead of yes.  I would put it inside the iteration so someone doesn't accidentally delete 100 dashboards.

Reply:
I made it so that it would ask for confirmation to delete per dashboard.